luigi-tutorials/tutorial-container.md

```diff
@@ -0,0 +1,321 @@
+
+    /**
+     * Constructor for a new <code>LuigiContainer</code>.
+     *
+     * @param {string} [sId] ID for the new control, generated automatically if no ID is given
+     * @param {object} [mSettings] Initial settings for the new control
+     *
+     * @extends sap.ui.core.webc.WebComponent
+     * @class
+     *
+     * The <code>sap.me.shared.LuigiContainer</code> is a component to render a Luigi micro frontend (iframe or web component based) without the need of being a Luigi Core application.
+     *
+     * @author SAP SE
+     *
+     * @constructor
+     * @public
+     * @alias sap.me.shared.LuigiContainer
+     */
+    var LuigiContainer = WebComponent.extend("luigi.ui5app.control.LuigiContainer", {
+        metadata: {
+            tag: "luigi-container",
+            properties: {
+                /**
+                 * Defines the view URL to load
+                 */
+                viewURL: {
+                    type: "string",
+                    defaultValue: "",
+                    mapping: {
+                        to: "viewURL"
+                    }
+                },
+
+                /**
+                 * Defines whether to load the Luigi component as web component
+                 */
+                webComponent: {
+                    type: "any", // boolean, but WebComponentRenderer renders out boolean values as empty attributes, which we have to fix in our attribute formatter
+                    defaultValue: false,
+                    mapping: {
+                        to: "webcomponent",
+                        formatter: "_mapIsWebComponent"
+                    }
+                },
+
+                /**
+                 * Defines the height of the control
+                 */
+                height: {
+                    type: "sap.ui.core.CSSSize",
+                    mapping: "style"
+                },
+
+                /**
+                 * Defines the width of the control
+                 */
+                width: {
+                    type: "sap.ui.core.CSSSize",
+                    mapping: "style"
+                },
+
+                /**
+                 * Defines label of the Luigi component
+                 */
+                label: {
+                    type: "string",
+                    defaultValue: ""
+                },
+
+                /**
+                 * Defines the context to use
+                 */
+                context: {
+                    type: "object",
+                    defaultValue: {}
+                },
+
+                theme: {
+                    type: "string",
+                    defaultValue: ""
+                }
+            },
+            methods: ["updateContext", "sendCustomMessage", "closeAlert"]
+        }
+    });
+
+    LuigiContainer.prototype._mapIsWebComponent = function(vIsWebComponent) {
+        return vIsWebComponent !== "false" && vIsWebComponent ? "true" : undefined;
+    };
+
+    /**
```

Comment:
same for the following comments => namespace! `luigi.ui5app.control.LuigiContainer`

Question in general => should we keep the JSDoc here? As it is a demo and not delivered as SAPUI5 code officially, we can also omit the JSDoc here and just put some comments, e.g. the functions can be commented in the methods array above...

Reply:
Thanks, I removed the extra comments and added your suggestion in [the latest commit.](https://github.com/alexandra-simeonova/drafts/pull/2/commits/a20e67a5eaea2606ccbe1c3ad68b7b34251fe099)

Reply:
See my comment above